ui/shared/Clipboard.jsx

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faClipboard } from '@fortawesome/free-regular-svg-icons';
+
+export default class Clipboard extends React.PureComponent {
+  static copyToClipboard(text) {
+    navigator.clipboard.writeText(text);
+  }
+
+  render() {
+    const { title, text } = this.props;
+
+    return (
+      <span
+        type="button"
+        className="pointer"
+        onClick={() => Clipboard.copyToClipboard(text)}
+      >
```

Comment:
Can you switch this out for a semantic button element? Either reactstrap's Button component or a button element.

Reply:
Also, I realize this code is existing but this seems like a strange way of calling the `copyToClipboard` method (as a static method). This component can be simplified to a functional component since we aren't keeping track of state.

Reply:
Done.